callback_plugins/custom_logger.py

```diff
@@ -54,11 +54,13 @@ def playbook_on_stats(self, stats):
 
     def log_task_result(self, host, result, task_name):
         # test_run_result.out only interested in the test tasks, not setup or debug.
-        if "RHELOSP" in task_name or "RHOSO" in task_name:
+        if "RHELOSP" in task_name or "RHOSO" in task_name  or "UI" in task_name:
```

Comment:
This works, but is not scalable.
I want to add a new config options to allow adding the prefix via a config option, either as an env var or as a line in the ansible.cfg.

This is okay for now, or you can try adding a new update for it yourself.
There is a well-defined pattern, and you can use this PR as a reference: [https://github.com/infrawatch/feature-verification-tests/pull/160 (ignore the non-config stuff](https://github.com/infrawatch/feature-verification-tests/pull/160/commits/b98eedd69cb2285baa0b0f82b7f0fe997f374d39)